runtime/syntax/help.vim

```diff
@@ -12,26 +12,37 @@ endif
 let s:cpo_save = &cpo
 set cpo&vim
 
+if !exists('g:help_example_languages')
+  let g:help_example_languages = #{ vim: 'vim' }
+endif
```

Comment:
Document this.
And, when setting g:help_example_languages, if you do not add vim, vim script syntax highlighting will not be available.

Reply:
> And, when setting g:help_example_languages, if you do not add vim, vim script syntax highlighting will not be available.

OK.  I will add it.

> Document this.

I think it is documented by `When not configured, help files support Vim script highlighting only`.

Reply:
yes should be fine